docs/installation/update.rst

```diff
@@ -1,6 +1,6 @@
-****************
-Upgrade to 1.5.0
-****************
+********************
+Upgrade to |version|
```

Comment:
Actually this is standard Sphinx! I used pbr to add a `|branch|` substitution so we can add that anywhere to dynamically set the branch anywhere in the docs.